pkg/kubelet/images/types.go

```diff
@@ -42,6 +42,9 @@ var (
 
 	// ErrInvalidImageName - Unable to parse the image name.
 	ErrInvalidImageName = errors.New("InvalidImageName")
+
+	// ErrInvalidSignature - Unable to validate the image signature.
+	ErrInvalidSignature = errors.New("InvalidSignature")
```

Comment:
Should this be more general to indicate that signature validation failed? Invalid signature could be a reason or it could be missing or not found on the registry.

Reply:
Changed it to `ErrSignatureValidationFailed`, does that fit better?